pkg/enforcer/enforcer.go

```diff
@@ -44,12 +49,33 @@ func (e *enforcer) Enforce(ctx context.Context, namespace string, resourceURI st
 	e.logger.Debugf("About to enforce resource '%s' in namespace '%s'", resourceURI, namespace)
 
 	// TODO: load namespace to look for annotations describing which attesters to enforce
-
+	config, err := rest.InClusterConfig()
```

Comment:
I prefer dependency injection for things like the kubernetes `clientset`.  Suggest moving the instantiation of the clientset into the `Controller` and then passing the instance in to the `New` function and then storing as a variable in the enforcer struct.

Reply:
Figured you'd prefer it somewhere else, I only put it in the function body because I wasn't sure where the best place to put it would be. Appreciate the guidance

Reply:
I've moved it to controller.Start but I thought it might be more appropriate in controller.New 

However, the error logging wouldn't match up styles with most other errors, including the recommendations on other sections of this PR. 

Should it be in controller.New instead though?

Reply:
agreed...`controller.Start` is right.